### PR TITLE
Jet air use fix and trails.

### LINKED ISSUE #48661: Inertia flying consume jetpack air.
BYOND Version: 513.1505
Server revision compiled on: 2020-01-07
Master commit: e87033d792cf4834f4fc18b0e6a2c2057859ce89

## Reproduction:

Equip jetpack turn on (with no stabilization) and jump to graveless room. 
You see as you fly doing nothing you jetpack use air.

### PR DESCRIPTION
**Pull request by Dennok**
## About The Pull Request
Remove continuous jet air using.
Remove jet air usung from carbon/Process_Spacemove and fix it in jet move_react.

## Why It's Good For The Game
Trails from nothing confuses.

Bugs is bad.
Jet that use air on every move and in gravity is bad.
fix #48661

## Changelog
:cl:
tweak: Jets make trails only on thrust
fix: jets use air only on thrust
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
Stop commenting shit out, this is not how you change things

**Comment by ninjanomnom** [comment]
You're removing the better way of handling this functionality, the code given is the kind of thing we don't want

**Comment by Dennok** [comment]
> You're removing the better way of handling this functionality, the code given is the kind of thing we don't want

How diferentes move from space drifting, pulling, throwing, and move from actual jet use?

**Comment by ninjanomnom** [comment]
I have no idea what you just wrote

**Comment by optimumtact** [comment]
>How diferentes move from space drifting, pulling, throwing, and move from actual jet use?

Best guess:
How do you differentiate moves from space drifting, pulling, throwing (i.e inertia) and moving from actual thrusting with the jet

**Comment by Dennok** [comment]
I want use jet when want move in space. 
Now jet use air on space drifting and i have zero ideas how to block jet use when i dont press buttons.

**Comment by ninjanomnom** [comment]
You could check if the client is holding a movement key down

File: code/game/objects/items/tanks/jetpack.dm [hunk 1]
```diff
@@ -17,6 +17,7 @@
 /obj/item/tank/jetpack/Initialize()
 	. = ..()
 	ion_trail = new
+	ion_trail.auto_process = FALSE
```
Comment (AnturK) [inline code/game/objects/items/tanks/jetpack.dm]: This looks unnecessary, you're doing everything move react.

File: code/modules/surgery/organs/augments_chest.dm [hunk 2]
```diff
@@ -132,6 +132,7 @@
 	. = ..()
 	if(!ion_trail)
 		ion_trail = new
+	ion_trail.auto_process = FALSE
```
Comment (AnturK) [inline code/modules/surgery/organs/augments_chest.dm]: Same thing

**Review by AnturK — CHANGES_REQUESTED** [review]

File: code/game/objects/items/tanks/jetpack.dm (hunk 1 above)
Comment (Dennok) [reply code/game/objects/items/tanks/jetpack.dm]: For what it need in autoprocessing when i manually call on demand?

https://github.com/tgstation/tgstation/blob/8358169b8e5a37508ae4f15ac1db007ef14a4a88/code/game/objects/effects/effect_system/effects_other.dm#L34-L41

**Comment by optimumtact** [comment]
antur did my man the dirty

**Pull request closed without merging**